### ISSUE TITLE
Promise.all like for Meteor.subscribe - #readyPromise()

### ISSUE THREAD

**Issue by guilhermedecampo** [deanius/meteor-promise#14]
Hello @deanius !

Thanks for the awesome package.

There is a way to use `Promise.all` like for readyPromise() ?

I have a few `this.subscribes` in my onCreated hook. 
Would like to wait them all to be ready so I can use the data.

**Comment by deanius** [comment]
Yes, if you have the subscription handles in an array, you can call `Promise.all` on an array of `readyPromise()` results. Something like this would help I think:

```
var subs = [];
Template.foo.onCreated( ()=> {
   subs.push( Meteor.subscribe("sub1") )
   subs.push( Meteor.subscribe("sub2") )
})
/* somewhere else */
Promise.all( subs.map( sub => sub.readyPromise() ) ).then( () => {
   //here you're subscribed to all
})
```

**Comment by guilhermedecampo** [comment]
Thanks @deanius !!

**Comment by deanius** [comment]
Sure, ask whatever you want @guilhermedecampo

**Comment by imdpk** [comment]
By using this technique, how can you make sure the data is stays reactive once changes are made in Mongo? It seems like a one time transaction. I am trying to use this technique with React and GetMeteorData Mixin.

**Comment by deanius** [comment]
Yes, @imdpk  - Promises are one-shot generally. So this can help time the initial load. I believe the mixin you mention handles re-rendering the component when Mongo changes but not sure- you can post a code sample if you want to discuss further.
